### PR TITLE
update init script

### PR DESCRIPTION
**Pull request by xu-cheng**
A small change since https://github.com/Homebrew/homebrew/pull/41034 is merged.

Also it would be nicer to transfer this wonderful project and `homebrew-aliases` to Homebrew Organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ First, tap this repository:
 
     brew tap bfontaine/command-not-found
 
-Then source the handler script in your `.bashrc`(bash) or `.zshrc`(zsh):
+Then add following line to your `.bashrc`(bash) or `.zshrc`(zsh):
 
-    eval "$(brew command-not-found-init)"
+    if brew command command-not-found-init > /dev/null; then eval "$(brew command-not-found-init)"; fi
```
Comment (bfontaine) [inline README.md]: Could you please use `&>` here? It’ll redirect both stdout *and* stderr to `/dev/null`.

Comment (xu-cheng) [inline README.md]: It's usually to print the error message. So users can be aware that some tool is missing. For example, `pyenv` and `rbenv` do the same thing.

Comment (bfontaine) [inline README.md]: Right.

**Comment by bfontaine** [comment]
Thank you @xu-cheng :)

**Pull request closed without merging**